package.json

```diff
@@ -21,6 +21,13 @@
   },
   "dependencies": {
     "@module-federation/utilities": "^1.7.0",
+    "@patternfly/patternfly": "^5.1.0",
+    "@patternfly/react-charts": "^7.3.0",
+    "@patternfly/react-core": "^5.1.1",
+    "@patternfly/react-icons": "^5.1.1",
+    "@patternfly/react-styles": "^5.1.1",
+    "@patternfly/react-table": "^5.1.1",
+    "@patternfly/react-tokens": "^5.1.1",
```

Comment:
@MariaAga do you need help packaging those?

Reply:
or will it be vendored inside `foreman-js` and we won't need packaging these separately?

Reply:
It will be inside foreman-js

Reply:
okay, so the change here is just temporarily until there is a foreman-js release out with those changes? and we don't package `tabbable`, so you *technically* don't need any packaging ACK ;-)
(GH will still want it, so, once there is a foreman-js release out, and the version requirement was bumped here, you get it)

Reply:
Yes, thanks